configure.ac

```diff
@@ -202,6 +202,9 @@ plugins/bookmarks/pluma-bookmarks.metainfo.xml.in
 plugins/Makefile
 plugins/codecomment/codecomment.plugin.desktop.in
 plugins/codecomment/Makefile
+plugins/wordcompletion/org.mate.pluma.plugins.wordcompletion.gschema.xml
+plugins/wordcompletion/wordcompletion.plugin.desktop.in
```

Comment:
It should be `plugins/wordcompletion/wordcompletion.plugin.desktop.in.in` as `wordcompletion.plugin.desktop.in` doesn't exist until you run configure.